### PR TITLE
Removed resourceTitle checks in maskinportenschema tests

### PR DESCRIPTION
**Pull request by ivartryti**
## Description
Removed resourceTitle checks in maskinportenschema tests

## Developer/Reviewer Checklist
- [ ] **Your** code builds clean without any errors or warnings
- [ ] No changes to config/appsettings or environment variables created in separate linked PR
- [ ] Manual testing done (required)
- [ ] Relevant integration test added (required for functional changes)
- [ ] Relevant automated test added (required for functional changes)
- [ ] All integration tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: test/K6/src/tests/maskinporten/maskinporten.js [hunk 1]
```diff
@@ -108,7 +108,6 @@ export function getMaskinPortenSchemaOfferedTest() {
     'get offered MaskinPortenSchemas - performedByUserId is ${performedByUserId}': (r) => r.json('0.performedByUserId') == org1.dagl.userid,
     'get offered MaskinPortenSchemas - coveredByOrganizationNumber is ${coveredByOrganizationNumber}': (r) => r.json('0.coveredByOrganizationNumber') == org2.orgno,
     'get offered MaskinPortenSchemas - resourceId is ttd-am-k6-nuf': (r) => r.json('0.resourceId') == appid,
-    'get offered MaskinPortenSchemas - resourceTitle is Maskinporten Schema - AM - K6 - NUF': (r) => r.json('0.resourceTitle.en') == 'Maskinporten Schema - AM - K6 - NUF',
     'get offered MaskinPortenSchemas - resourceType is MaskinportenSchema': (r) => r.json('0.resourceType') == 'MaskinportenSchema',
```
[the remark this answers is not in the document]
Comment (jonkjetiloye) [reply test/K6/src/tests/maskinporten/maskinporten.js:111]: Updating paths should  be done for all MaskinportenSchema tests in accordance with:
https://github.com/Altinn/altinn-access-management/pull/402

As the old paths are to be removed

[the remark this answers is not in the document]
Comment (ivartryti) [reply test/K6/src/tests/maskinporten/maskinporten.js]: updated the endpoints and removed two obsolete asserts

File: test/K6/src/tests/maskinporten/maskinporten.js [hunk 2]
```diff
@@ -108,8 +108,6 @@ export function getMaskinPortenSchemaOfferedTest() {
     'get offered MaskinPortenSchemas - performedByUserId is ${performedByUserId}': (r) => r.json('0.performedByUserId') == org1.dagl.userid,
```
Comment (jonkjetiloye) [inline test/K6/src/tests/maskinporten/maskinporten.js:108]: Can also add verification of all properties of the response model. Currently missing:
offeredByName
offeredByOrganizationNumber

**Review by jonkjetiloye — APPROVED** [review]

File: test/K6/src/tests/maskinporten/maskinporten.js (hunk 2 above)
Comment (ivartryti) [reply test/K6/src/tests/maskinporten/maskinporten.js:109]: added these checks

**Pull request merged**